egg/zoo/compo_vs_generalization/curriculum_games.py

```diff
@@ -87,6 +90,15 @@ def forward(self, sender_input, labels, receiver_input=None, aux_input=None):
             probs[:,  -1] = 1.
         elif self.reveal_distribution == 'uniform':
             probs = torch.ones((batch_size, self.curriculum_level)) / self.curriculum_level
+        elif self.reveal_distribution == 'natural':
+            # this distribution simulates sampling from a dataset containing 
+            # every masked input (up to curriculum_level masks)
+            
```

Comment:
Maybe worth adding a quick line explanining that there are (n choose k) x N_v^k elements with k masks